tools/gfx_tools/compose.py

```diff
@@ -179,6 +187,8 @@ def verify(self):
             if pngnum and pngname not in self.referenced_pngnames:
                 print("image filename '{}' index '{}'".format(pngname, pngnum) +
                       " was not used in any tile_config.json entries")
+                global error_logged
+                error_logged = True
```

Comment:
This is too broad to be considered an error. Consider fillers:
```
parsing tilesheet normal.png
parsing filler tilesheet fillerhoder.png
skipping filler for vp_door
<...>
image filename '1872_vp_door_0' index '2479' was not used in any tile_config.json entries
```
Because there is *vp_door* in `normal.png`, *vp_door* from `fillerhoder.png` becomes unused, hence this error(?) is printed.

A better solution, of course, would be to omit unused images and don't include them in tilesheets at all, but I understand that this is out of scope for this PR, so I suggest to consider this print as a warning and not an error.

Reply:
Yeah that makes sense to me, will fix.

Reply:
it's really more accurate to say that the sprite at normal_32x32/1000_vp_door.png fills the requirement for vp_door, so the sprite at fillerhoder/1872_vp_door_0.png is unused.

That is not necessarily a bug, because the point of filler folders is to contain sprites that will not be used if the non-filler folders provide better sprites.  But there's no easy way in verify() to detect that a png is in a filler directory, not a normal directory, and have an unreferenced image in a non-filler directory *usually* means there is a bug.

At any rate, unused images are not included in tilesheets as it is.

So I don't think this should be an error, but I also don't think that the message should be removed because it can point out an error.

Probably the easiest (hah!) solution would be to add `self.filler_pngnames = []` to `class PngRefs`, add `refs.filler_pngnames.append(pngname) if refs.filler` in walk_dirs, and then condition the message and error logging on `pngname not in self.filler_pngnames`

but someone may want to double-check that logic, because walk_dirs() is already supposed to catch pngnames from filler tilesheet directories.